mentors/java/a.json

```diff
@@ -22,5 +22,13 @@
     "link_url": "https://www.dericbourg.net",
     "avatar_url": "http://www.dericbourg.net/images/profile.png",
     "bio": "Software engineer for 10 years now, mostly experienced on the JVM (Java and Scala) for web facing systems. Eager to learn, you'll probably find me trying new things on the other tracks!"
+  },
+  {
+    "github_username": "m4ver1k",
```

Comment:
Hello and thank you for adding your bio. Unfortunately, it has been saved in the wrong format.  Based on your GitHub user name it should be in the format: mentors/java/m.json 
Can you make the change and update your PR. Thank you.